### ISSUE TITLE
Unable to load CMap data

### ISSUE THREAD

**Issue by iditkosti** [bhklab/PharmacoGx#99]
Unable to load the full CMap data (just the CMAPsmall example)

**Comment by ChristopherEeles** [comment]
Hi @iditkosti,

We added a parameter to `availablePSets()` called `canonical` which defaults to `TRUE`. This filters to only show PharmacoSets for we have deemed to sufficiently curated and reproducible to meet the highest data quality standards.

CMAP is not a canonical PSet, thus it does now show by default. To view it, use;
```
availablePSets(canonical=FALSE)
```

In `downloadPSet` all PSets are available, you just need to specify the 'PSet Name' value from `availablePSets`. In this case to download CMAP, you must use:
```
CMAP <- downloadPSet("CMAP_2016")
```

I will review our PSet creation documentation and try to make this more clear. We haven't modified that vignette in a few versions.

Sorry for any confusion.

Best,
Christopher Eeles
Software Developer
BHK Lab | PM-Research | UHN
